src/FP/Abstractions/Applicative.agda

```diff
@@ -1,15 +1,14 @@
 {-# OPTIONS --without-K --exact-split --safe --no-unicode #-}
```

Comment:
Interesting that you use `--no-unicode`.  Just curious, why do you not like unicode?

Reply:
Otherwise, those pragmas are exactly the ones I use in my agda-algebras library. 👍🏼 

By the way, I think they're maybe phasing out `--without-K` in favor of `--cubical-compatible`.  (I much prefer the old name as it seems to give a more precise description of what going on.)

Reply:
Drop pragma but awoid.